deviceplugin/generic.go

```diff
@@ -69,44 +65,13 @@ type Group struct {
 	// When the paths have differing cardinalities, that is, the globs match different numbers of devices,
 	// the cardinality of each path is capped at the lowest cardinality.
 	Paths []*Path `json:"paths"`
+	// UsbSpecs is the list of USB specifications that this device group consists of.
```

Comment:
For consistency let's keep all Usb->USB
```suggestion
	// USPSpecs is the list of USB specifications that this device group consists of.
```

Reply:
Resolved 3927c05